src/effects/VST/VSTEffect.cpp

```diff
@@ -2191,18 +2188,14 @@ float VSTEffectWrapper::callGetParameter(int index) const
    return mAEffect->getParameter(mAEffect, index);
 }
 
+
 void VSTEffect::callSetParameter(int index, float value)
 {
    bool setOK = callSetParameterB(index, value);
-
-   if (setOK)
-   {
-      for (const auto& slave : mSlaves)
-         slave->callSetParameterB(index, value);
-   }
```

Comment:
Your step 5.1 removes these lines.  I already explained how it should be done in step 4, eliminating the need for two versions of the set-parameter function.